### PR TITLE
apply change in #133 to built medium.js

### PR DESCRIPTION
**Pull request by levity**
The change in #133 wasn't applied to the built medium.js and medium.min.js, which makes the repo unusable via NPM for those who are using the master branch and need that fix.

This PR just applies it to the unminified file, because I don't have yui-compressor set up locally and don't use the minified file, so it's not technically complete. But it should be pretty easy for someone else to do that, so I figured I would create this PR to draw attention to the issue.

### CODE REVIEW

File: medium.js [hunk 1]
```diff
@@ -1784,7 +1784,7 @@ Medium.defaultSettings = {
 				}
 
 				while (html.length > 0) {
-                                        parent.insertBefore(html[html.length - 1], wedge);
+          parent.insertBefore(html[0], wedge);
```
Comment (nichochar) [inline medium.js]: You should probably try and match the indentation above

**Comment by robertleeplummerjr** [comment]
Also, this conflicts.  Can you resolve them and well implement?

**Comment by levity** [comment]
I'll just close this one since master has moved on quite a bit.

**Comment by levity** [comment]
i see now that #167 already covered this, actually

**Pull request closed without merging**